site/en/tutorials/sequences/text_generation.ipynb

```diff
@@ -819,14 +819,14 @@
     "* The hidden state returned by the model is fed back into the model so that it now has more context, instead than only one word. After predicting the next word, the modified hidden states are again fed back into the model, which is how it learns as it gets more context from the previously predicted words.\n",
     "\n",
     "\n",
-    "![](https://drive.google.com/uc?export=view&id=1MeU-bnClsvknBWTCCPgbrCyGRoyaZf9k)\n",
+    "![](https://drive.google.com/uc?export=view&id=1ZdZmGCfti_QhBrmg06teYhO2-zKl-fMV)\n",
```

Comment:
Can you add these images to the git repo?
Just create an images directory in the collection, ex: /tutorials/sequences/images/

Reply:
Done! Thank you!